adapter-cqld4/src/main/java/io/nosqlbench/cqlgen/model/CqlKeyspaceDef.java

```diff
@@ -55,12 +55,7 @@ public String getName() {
 
     @Override
     public String toString() {
-        return "CqlKeyspace{" +
-            "keyspaceName='" + keyspaceName + '\'' +
-            ", stats=" + stats +
-            ", isDurableWrites=" + isDurableWrites +
-            ", replicationData='" + replicationData + '\'' +
-            '}';
+        return STR."CqlKeyspace{keyspaceName='\{keyspaceName}\{'\''}, stats=\{stats}, isDurableWrites=\{isDurableWrites}, replicationData='\{replicationData}\{'\''}\{'}'}";
```

Comment:
Commented earlier. Adding here for tracking

Reply:
fixed